api/core/model_runtime/model_providers/moonshot/llm/moonshot-v1-8k.yaml

```diff
@@ -21,6 +21,17 @@ parameter_rules:
     default: 512
     min: 1
     max: 8192
+  - name: response_format
+    label:
+      zh_Hans: 回复格式
+      en_US: response_format
+    type: string
+    help:
+      zh_Hans: 指定模型必须输出的格式
+      en_US: specifying the format that the model must output
+    required: false
+    options:
```

Comment:
I think it would be better to have `text` as well.

Reply:
Ok，add text now。
![image](https://github.com/user-attachments/assets/c876c321-571b-4852-bf1a-edb35080e059)